### PR TITLE
Reader post comments: show first comment

### PR DESCRIPTION
**Pull request by ScoutHarris**
Ref: #17511 
Depends on:  #17602

The first comment is now displayed in the Reader post detail Comments table.

Note: the `postDetailsComments` feature flag is enabled for develop.

To test:
- Go to Reader > post > details.
- Verify:
  - The first comment is displayed in the Comments table.
  - All the comment actions are hidden (like, reply, moderation, accessory button)

<kbd>comment</kbd>

## Regression Notes
1. Potential unintended areas of impact
N/A. WIP.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. WIP.

3. What automated tests I added (or what prevented me from doing so)
N/A. WIP.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift [hunk 1]
```diff
@@ -207,6 +207,16 @@ class CommentContentTableViewCell: UITableViewCell, NibReusable {
         webView.isOpaque = false // gets rid of the white flash upon content load in dark mode.
         webView.loadHTMLString(formattedHTMLString(for: comment.content), baseURL: Self.resourceURL)
     }
+
+    /// Hide all actions for the comment.
+    ///
+    func hideAllActions() {
```
Comment (dvdchr) [inline WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift]: If this is called before `configure`, the values here would be overridden. Although this is only used in one place, how about adding a note here to inform that this needs to be called after `configure`?

**Review by dvdchr — APPROVED** [review]
Works as expected. Left one minor comment, and also found a minor visual bug. Looks like the content cell should be aligned to the leading edge of the button to follow the designs:

Simulator Screen Shot - iPhone 12 - 2021-12-01 at 21 36 16

Other than that, LGTM! 👍

File: WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift (hunk 1 above)
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift]: Done!

**Comment by ScoutHarris** [comment]
> Looks like the content cell should be aligned to the leading edge of the button to follow the designs

🤦 Thank you. It doesn't look like a quick fix, so I'll do it separately.

**Pull request merged**